www/front_src/src/Header/SwitchThemeMode/index.tsx

```diff
@@ -1,104 +1,73 @@
+/* eslint-disable hooks/sort */
+import { useTransition, useState } from 'react';
+
 import { equals } from 'ramda';
 import { useAtom } from 'jotai';
 import { useLocation } from 'react-router-dom';
+import clsx from 'clsx';
 
-import { styled } from '@mui/material/styles';
-import Switch from '@mui/material/Switch';
+import { Switch, ListItemText } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
+import { grey } from '@mui/material/colors';
 
 import { userAtom, ThemeMode } from '@centreon/ui-context';
 import { patchData, useRequest } from '@centreon/ui';
 
-import svgSun from './images/sun.svg';
-import svgMoon from './images/moon.svg';
-
-interface StyleProps {
-  darkModeSvg?: string;
-  lightModeSvg?: string;
-}
-
-const ThemeModeSwitch = styled(Switch, {
-  shouldForwardProp: (prop) =>
-    !equals(prop, 'color') &&
-    !equals(prop, 'lightModeSvg') &&
-    !equals(prop, 'darkModeSvg'),
-})<StyleProps>(({ theme, darkModeSvg, lightModeSvg }) => ({
-  '& .MuiSwitch-switchBase': {
+const useStyles = makeStyles((theme) => ({
+  container: {
+    '& .MuiSwitch-thumb': {
+      backgroundColor: 'white',
+    },
+    '& .MuiSwitch-track': {
+      backgroundColor: '#aab4be',
+      opacity: 1,
+    },
+    alignItems: 'center',
+    display: 'flex',
+  },
+  containerMode: {
+    display: 'flex',
+    justifyContent: 'space-around',
+  },
+  containerSwitch: {
     '&.Mui-checked': {
-      '& + .MuiSwitch-track': {
-        backgroundColor: '#aab4be',
-        opacity: 1,
-      },
-      '& .MuiSwitch-thumb:before': {
-        backgroundImage: `url(${darkModeSvg})`,
+      '&:hover': {
+        backgroundColor: 'unset',
       },
-      color: 'transparent',
-      transform: 'translate(15px,-50%)',
     },
     '&:hover': {
-      backgroundColor: 'transparent',
+      backgroundColor: 'unset',
     },
-    color: 'black',
-    margin: 0,
-    position: 'absolute',
-    top: '50%',
-    transform: 'translate(-0.5px,-50%)',
   },
-  '& .MuiSwitch-thumb': {
-    '&:before': {
-      backgroundImage: `url(${lightModeSvg})`,
-      backgroundPosition: 'center',
-      backgroundRepeat: 'no-repeat',
-      content: "''",
-      height: '100%',
-      left: theme.spacing(0),
-      position: 'absolute',
-      top: theme.spacing(0),
-      width: '100%',
-    },
-    backgroundColor: 'white',
-    height: theme.spacing(3),
-    width: theme.spacing(3),
+  disabledMode: {
+    color: grey[600],
```

Comment:
You can use the default disabled color from the palette

Reply:
if i use palette.action.disabled i get : 
![darkMode](https://user-images.githubusercontent.com/97687698/173795685-50c484a4-340e-4a43-8e56-5130dcb637d5.PNG)

Reply:
Can you try the palette.text.disabled, please?

Reply:
it's the same thing transparent background is not displayed because of the color black, but  I used a standard color mui 
 and adding a value opacity ^^